kate_brownbill_dungon_fighters_v1.0.py

```diff
@@ -0,0 +1,838 @@
+    f3c = Fighter([FIREBOLT, MAGIC_SWNG, SHIELD_BASH], "Fire Knight", "A greater elementle who works as a knight. Most demons see her as a lesser being.", 5, 5, 3)
+    t3 = (p3, [f3a, f3b, f3c])
+    
+    #The Imp Lord
+    p4 = Player(1, "The Imp Lord", "The imp lord, here to defeat you once and for all.")
+    f4a = Fighter([MAGIC_SWNG, FRWD_SWNG, MAGIC_RFCT, FIREBOLT], "Dar", "The lord of imps, the ruler of hell.", 7, 5, 4)
+    t4 = (p4, [f4a])
+    
+    #area details
+    area_name = "Hell"
+    begin_text = "Passing through the portal to hell, the landscape changes durasticly. The sky is a chaotic mess of purples and greens. The hot, dry air leaves you constently thirsty."
+    end_text = "As Dar the demon lord falls to the ground, you open the door to the inner sanctum where a large crystle poweres the rift between worlds. Distroying it would close the rift and save your world, but it would trap you here. Eve urges you to distroy it, Magios interupts her. The whole party erupts in argument."
+    new_fighter_txt = "While your party is arguing, an Echolo tugs at your shirt. 'Hey uh...I have a friend who wanted to-' an implike jester came through the door with a big smile. 'Fellas, fellas...we can make a compramise here. I'll tell you this, I'm one of the most powerful demons in hell. If I tellaport you home, after you shut down the portle, I won't ask much of you. All I'd ask for is for entertainment. Anytime you wish, you can take my chalenge, at no risk you yourself even! So, what do you say?'"
+    af = Fighter([MAGIC_SWNG, FRWD_SWNG, FIREBOLT], "Echa", "An echolo who wants to explore the mortle world.", 4, 4, 2)
+    
+    #return time
+    area_info = (area_name, begin_text, end_text, new_fighter_txt, af, [t1, t2, t3, t4])
+    return area_info
+    
+            
+def jester_game():
+    player = Player(0 ,"JESTER", "IT'S ME, IT'S YOU")
+    him = Fighter([FIREBOLT, FRWD_SWNG, MAGIC_SWNG, SHIELD_BASH, MAGIC_RFCT], "The Demonic Joker", "A sub-speaces of imp who is arguably the most powerful being in existence", 6, 6, 6)
+    lv = get_menu(["1", "2", "3", "4"], "Pick a dificalty level.\n[1] [2] [3] [4]\n", "That's not a valid levle.")
+    foe = Player(4, "THE ENEMY", "BORING")
+    team = []
+    for i in range(lv + 1):
+        black_moves = []
+        red_moves = []
+        while black_moves == red_moves:
+            black_moves = create_moves(4)
+            red_moves = create_moves(4)      
+        card_black = Fighter([MAGIC_RFCT, FRWD_SWNG, SHIELD_BASH, MAGIC_SWNG], "Dark Card", "Woven from shadow. Does it even exist?", 3 + i, 3 + i, 1 + i)
+        card_red = Fighter([MAGIC_RFCT, FIREBOLT, MAGIC_SWNG, SHIELD_BASH], "Red Card", "Woven from blood. Does it even exist?", 3 + i, 4 + 1, 0 + 1)
+        team.append(card_black)
+        team.append(card_red)
+    do_battle([him], team, player, foe)
+            
+menu = True
```

Comment:
Python programs usually use main for the entry point to the program...  i.e.,

https://www.guru99.com/learn-python-main-function-with-examples-understand-main.html

That makes sure that the variables like menu and jester are limited to just the main program,  which makes the code easier to understand.